src/pattern_generator.cpp

```diff
@@ -448,6 +450,19 @@ void PatternGenerator::enableCgSettings(bool en)
 	colour_button_low->setEnabled(en);
 }
 
+bool PatternGenerator::getUse_decoders() const
```

Comment:
I think it is a good idea not to mix two naming conventions (camel-case and snake-case). Also, the rest of the code is mostly written with camel-case. Keeping things consistent makes code easier to read.